### ISSUE TITLE
代码疑问

### ISSUE THREAD

**Issue by dingandchen** [nuliweixiao/EFC#3]
x_1_map = out_1.reshape(N, 1, -1)
mean_1 = x_1_map.mean(dim=2, keepdim=True)
x_1_av = x_1_map / mean_1
  x_2_2 = F.softmax(x_1_av, dim=1)
这里softmax的维度为1但是形状为n ，1，c*h*w不就没意义了吗
我测试了输出的掩码结果均为1

**Comment by wap1024** [comment]
import sparse_conv，请问您这边安装了吗？

**Comment by userfirst001** [comment]
请问，代码是怎么去运行，小白没看懂
